src/MLJInterface.jl

```diff
@@ -155,7 +153,7 @@ MLJModelInterface.@mlj_model mutable struct LGBMClassifier <: MLJModelInterface.
     objective::String = "multiclass"::(_ in CLASSIFICATION_OBJECTIVES)
```

Comment:
Can't comment against the commented out sigmoid param hence commenting here (see lines above for sigmoid). I'm not quite sure about adding sigmoid here because of this: https://github.com/IQVIA-ML/LightGBM.jl/blob/master/src/MLJInterface.jl#L150-L152. Since the defaulted objective on the MLJ classifier is multiclass, then adding a defaulted sigmoid here might cause some issues but on the other hand the LGBMClassification also is defaulted to multiclass and still has the sigmoid defaulted to 1.

Reply:
Re-enabled `sigmoid` with a default value since we have several other parameters which work only with `binary` or `multiclassova` such as `scale_pos_weight` or `is_unbalance`. Their defaults have no impact with the currently default objective set to `multiclass` but are made available if the user changes the objective and wants to use them.